### ISSUE TITLE
If inline style is applied to any react components I get an error !!!

### ISSUE THREAD

**Issue by antoniopangallo** [boopathi/react-svg-loader#205]
Example:
```
const style = { backgroundColor: red}
<div style={style} />
```

That will trigger an error 

> Cannot read property 'split' of undefined at cssToObj .../node_modules/babel-plugin-react-svg/lib/css-to-obj.js:9:21

This is the css-to-obj.js

```
export default function cssToObj(css: string) {
  let o = {};
  let elements = css.split(";");
  elements.filter(el => !!el).map(el => {
    let s = el.split(":"),
      key = s.shift().trim(),
      value = s.join(":").trim();
    o[key] = value;
  });
  return o;
}
```
My solution 
```
export default function cssToObj(css: string) {
  let o = {};
  if (typeof css !== "undefiend") {
   let elements = css.split(";");
  elements.filter(el => !!el).map(el => {
    let s = el.split(":"),
      key = s.shift().trim(),
      value = s.join(":").trim();
    o[key] = value;
  });
 }
  return o;
}
```

**Comment by GiancarlosIO** [comment]
i have the same problem using the babel-plugin-react-svg

**Comment by boopathi** [comment]
closed by #206
